src/components/home/FavouriteEntries/index.js

```diff
@@ -70,6 +71,16 @@ export class FavouriteEntries extends PureComponent /*:: <Props> */ {
                   <EntryCard entry={f} key={f.metadata.accession} />
                 ))}
               </AnimatedEntry>
+              <Link
```

Comment:
Not sure if we should show this.... or maybe we can add a validation to only show if there are differences.